Homework3/Homework3/Program.cs

```diff
@@ -0,0 +1,238 @@
+                                 + $"Register{Environment.NewLine}"
+                                 + $"Exit{ Environment.NewLine}");
+
+                string command = PromptUserForInput("command");
+
+                if (command.Equals("login", StringComparison.OrdinalIgnoreCase))
+                {
+                    LoginUser(filePath);
+                }
+
+                if (command.Equals("register", StringComparison.OrdinalIgnoreCase))
+                {
+                    RegisterUser(filePath);
+                }
+
+                if (command.Equals("exit", StringComparison.OrdinalIgnoreCase))
+                {
+                    Environment.Exit(0);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Attempts to log the user in using their user name and password. 
+        /// Prints a message to the console if successful or unsuccessful.
+        /// </summary>
+        /// <param name="filePath">The file of user data.</param>
+        static void LoginUser(string filePath)
+        {
+            string userName = PromptUserForInput("user name");
+            string password = PromptUserForInput("password");
+
+            if (CheckIfCredentialsAreValid(filePath, userName, password))
+            {
+                Console.WriteLine($"Hello!{Environment.NewLine}");
+            }
+            else
+            {
+                Console.WriteLine($"Invalid credentials.{Environment.NewLine}");
+            }
+        }
+
+        /// <summary>
+        /// Prompts the user for input.
+        /// </summary>
+        /// <param name="inputType">The type of data being prompted for.</param>
+        static string PromptUserForInput(string inputType)
+        {
+            Console.Write($"Enter a {inputType}: ");
+            string input = Console.ReadLine();
+
+            return input;
+        }
+
+        /// <summary>
+        /// Checks the specified file to see if the credentials are valid.
+        /// </summary>
+        /// <param name="filePath">The file being checked against.</param>
+        /// <param name="userName">The user name being checked.</param>
+        /// <param name="password">The password being checked.</param>
+        /// <returns>
+        /// True if the user name and password match, false if it does not match.
+        /// </returns>
+        static bool CheckIfCredentialsAreValid(string filePath, string userName, string password)
+        {
+            StreamReader reader = new StreamReader(filePath);
+
+            bool isValid = false;
+
+            while (reader.Peek() >= 0)
+            {
+                string line = reader.ReadLine();
+                string[] words = line.Split(",");
+
+                isValid = words[0].Equals(userName) && words[1].Equals(password);
+
+                if (isValid)
+                {
+                    break;
+                }
+            }
+
+            reader.Close();
```

Comment:
Prefer to call `Dispose` over `Close` for unamanged resources. Also, prefer to wrap it in a `using(var reader = new StreamReader(path)` block. In your case, closing is the same as disposing.